src/main/java/com/gooddata/account/AccountService.java

```diff
@@ -49,4 +48,45 @@ public void logout() {
             throw new GoodDataException("Unable to logout", e);
         }
     }
+
+    /**
+     * Creates new account in given organization (domain)
+     * @param account to create
+     * @param organizationName (domain) in witch account should be created
+     * @return new account
+     */
+    public Account createAccount(Account account, String organizationName) {
+        notNull(account, "account");
+        notNull(organizationName, "organizationName");
+
+        try {
+            final UriResponse uriResponse = restTemplate.postForObject(Account.CREATE_URI, account, UriResponse.class, organizationName);
+            return getAccount(Account.getId(uriResponse.getUri()));
+        } catch (GoodDataException | RestClientException e) {
```

Comment:
according to documentation there are no error states... I knout only about validation and access rights errors